users/views.py

```diff
@@ -47,3 +47,22 @@ def post(self, request):
         except ValidationError: 
             return JsonResponse({'message' : 'INVALID_USER'}, status=400)
 
+class SignInView(View):
+    def post(self, request):
+        try:
+            data          = json.loads(request.body)
+            user_email    = data['email']
+            user_password = data['password']
+            user          = User.objects.get(email = user_email)
+            
+            if not bcrypt.checkpw(user_password.encode('utf-8'), user.password.encode('utf-8')): 
+                return JsonResponse({'message' : 'INVALID_PASSWORD'}, status=401)
```

Comment:
로그인 실패 시 아이디가 틀렸는지, 비밀번호가 틀렸는지 모르게 하시는 것 이 좋습니다.
```suggestion
                return JsonResponse({'message' : 'INVALID_USER'}, status=401)
```

Reply:
넵 수정하겠습니다!